tests/config/test_trainer_config.py

```
@@ -0,0 +1,206 @@
+
+
+def test_wandb_config():
+    """wandb_config tests.
+
+    Check default values and customization
+    """
+    # Check default values
+    conf = OmegaConf.structured(WandBConfig)
+    assert conf.entity is None
+    assert conf.project is None
+    assert conf.wandb_mode == None
```

Ruff (0.8.2)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)